src/01-functions.js

```diff
@@ -39,7 +45,11 @@ function greet() {}
  *  add(10, 20);
  *  //> 30
  */
-function add() {}
+
+function add(a,b) {
+ let result = a + b;
```

Comment:
Here declaring a variable is not necessary.  This is a fine practice to use now but down the road we should start to think about when we can directly return the result of an operation.  ` return a + b ` is the same functionally.